### ISSUE TITLE
`SharedPtrAdapter` isn't accept `shared_ptr` in any case

### ISSUE THREAD

**Issue by unit-a-user** [bytedeco/javacpp#577]
Everytime, when I used `@SharedPtr` as return value, I getting compile error:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	C7568	argument list missing after assumed function template 'SharedPtrAdapter'	YAV	C:\VULKAN\YetAnotherVulkanMod\src\main\cpp\jniAlter.cpp	907	
```
i.e. `SharedPtrAdapter` isn't accept `std::shared_ptr`.

**Comment by unit-a-user** [comment]
Also I see, that `SHARED_PTR_NAMESPACE` not defined, despite of should to be `std`.

**Comment by saudet** [comment]
Yeah, maybe it should `std` by default when it detects a C++11 compiler.

**Comment by saudet** [comment]
The fix for this has been released with JavaCPP 1.5.8. Enjoy!
